### PR TITLE
added info for hwk2

### PR DESCRIPTION
**Pull request by ctcaulfield**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,9 +66,15 @@ People
 * [Stephen Bongner](people/stephen-bongner.md) - sb2483 - [http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080](http://ec2-13-59-91-251.us-east-2.compute.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/sb2483/skeleton.svg?style=svg)](https://circleci.com/gh/sb2483/skeleton)
 * [Cameron Boroumand](people/cameron-boroumand.md) - cb596 - [http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html](http://cameronboroumand.com.s3-website.us-east-2.amazonaws.com/grading.html)
 * [Mikaela Brown](people/mikaela-brown.md) - mb2587 - [https://s3.amazonaws.com/aws-website-mb-u6ygo/grading.html](https://s3.amazonaws.com/aws-website-mb-u6ygo/grading.html)
+<<<<<<< HEAD
```
Comment (amfleming) [inline README.md]: Yikes!   What is this?

File: README.md [hunk 2]
```diff
@@ -66,9 +66,15 @@ People
...
+* [Yuan Chang](people/yuan-chang.md) - yc2378 - [https://s3.amazonaws.com/yuanchang.com/grading.html](https://s3.amazonaws.com/yuanchang.com/grading.html)
+=======
 * [Christopher Caulfield](people/christopher-caulfield.md) - ctc98 - [http://www.caulfield.io.s3-website-us-east-1.amazonaws.com/grading.html](http://www.caulfield.io.s3-website-us-east-1.amazonaws.com/grading.html)
 * [Gaurav Chaturvedi](people/gaurav-chaturvedi.md) - gkc24 - [http://ec2-34-229-141-246.compute-1.amazonaws.com:8080](http://ec2-34-229-141-246.compute-1.amazonaws.com:8080)[![CircleCI](https://circleci.com/gh/gauravChaturvedi/skeleton/tree/master.svg?style=svg)](https://circleci.com/gh/gauravChaturvedi/skeleton/tree/master)
 * [Yuan Chang](people/yuan-chang.md) - yc2378 - [http://34.231.71.205:8080][![CircleCI](https://circleci.com/gh/JennieYuanChang/skeleton.svg?style=svg)](https://circleci.com/gh/JennieYuanChang/skeleton)
+>>>>>>> CT-CS5356-Fall2017/master
```
Comment (amfleming) [inline README.md]: no no no

**Review by amfleming — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (rchatterjee) [inline README.md]: These are conflict files. You must remove these. You can check out [thes](https://help.github.com/articles/resolving-a-merge-conflict-using-the-command-line/) for how to resolve conflicts in git.

**Review by rchatterjee — COMMENTED** [review]
You need to resolve the conflicts.

File: README.md (hunk 1 above)
Comment (ctcaulfield) [reply README.md]: hm, not sure how that happened. let me try putting it up again

**Comment by ctcaulfield** [comment]
should be good now @amfleming

**Pull request merged**